lib/strategies/accept_header_strategy.rb

```diff
@@ -0,0 +1,42 @@
+module Phenotype
+  class AcceptHeaderStrategy
+    attr_reader :env, :content_type, :version_key
+
+    def initialize(app_name: 'rentpath',
+                   version_key: 'version', 
+                   format: 'json')
+      @version_key = version_key
+      @content_type = build_content_type(app_name, format)
+    end
+
+    def get_version(env)
```

Comment:
So methods with `get` and `set` in their name are typically not considered idiomatic Ruby. Instead, it would be more idiomatic to just call `#version`.

Have you considered passing in the `env`object to the initializer instead of the method that retrieves the version? Perhaps something like:

```ruby
module Phenotype
  class AcceptHeaderStrategy
    def initialize(env, mime_type = "application/vnd.rentpath.api+json") do
      @env, @mime_type = env, mime_type
    end

    def version
      env['HTTP_ACCEPT'].match(/#{Regexp.quote(mime_type)};version=(?<version>\d+)/) { |match| match[:version] }
    end
  end
end
```